packages/components/inputs/localized-multiline-text-input/src/translation-input.styles.ts

```diff
@@ -25,18 +26,27 @@ const getTextareaStyles = (props: TTranslationInputStylesProps) => {
   return baseStyles;
 };
 
+const getLanguageLabelBackgroundColor = (
+  props: TTranslationInputStylesProps
+) => {
+  if (props.isDisabled)
+    return designTokens.backgroundColorForLocalizedMultilineTextInputLabelWhenDisabled;
+  return designTokens.backgroundColorForLocalizedMultilineTextInputLabel;
+};
+
```

Comment:
**nit**: I think not using curly braces to make code less readable. In this case, perhaps it would be easier to follow if we use a ternary.

```suggestion
const getLanguageLabelBackgroundColor = (
  props: TTranslationInputStylesProps
) => props.isDisabled
  ? designTokens.backgroundColorForLocalizedMultilineTextInputLabelWhenDisabled
  : designTokens.backgroundColorForLocalizedMultilineTextInputLabel;

```

Reply:
Thanks. This was addressed here => [45d2402](https://github.com/commercetools/ui-kit/pull/2374/commits/45d240294657cd5de9fe27d01a18344998580347)